### PR TITLE
VWRs troubleshooting

### PR DESCRIPTION
**Pull request by diana-macrometa**
Add VWRs Troubleshooting content.

### CODE REVIEW

File: docs/photoniq/vwrs/troubleshoot-vwrs.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 120
+title: Troubleshoot VWRs
+---
+
+This page provides troubleshooting information for Macrometa PhotonIQ Virtual Waiting Rooms (VWRs).
+
+## No Waiting Room Configured
+
+If a waiting room has not been configured, then the waiting room library returns the header _No-Waiting-Room_, which indicates no waiting room was configured. This means that the domain URL is not configured correctly.
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/troubleshoot-vwrs.md]: This sounds strage:  "... waiting room has not been configured .... no waiting room was configured"  Is that ok?

File: docs/photoniq/vwrs/troubleshoot-vwrs.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Troubleshoot Waiting Room HTML
+
+You can configure the cloud origin (e.g. NetStorage) path to the waiting room HTML through the library or the domain record. To preview this HTML, open your preferred browser and navigate to the following URL: `https://<hostname>/<waiting-room-path>/preview`
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/troubleshoot-vwrs.md]: I am confused on this.  "cloud origin path" and the netstorage reference.
@rahul-naik1 The last sentence makes sense.  But the first part of the sentence is confusing. 
"... the waiting room HTML through the library or domain record." makes sense.  But I am struggling with the first part of the sentence.  Can you clarify this part...

File: docs/photoniq/vwrs/troubleshoot-vwrs.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+(for example, `http://show.store.com/checkout/preview`).  
+
+## Troubleshoot Waiting Room Cookie
+
+The _vwrs-cookie_ cookie holds valuable information regarding each user's access to the waiting room. This information is securely encrypted using the library encryption key.
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/troubleshoot-vwrs.md]: @rahul-naik1 would it make sense to mention what the variable for the encryption key?  But way, how is this now set using the npm package?

**Review by joubert-macrometa — CHANGES_REQUESTED** [review]

File: docs/photoniq/vwrs/troubleshoot-vwrs.md (hunk 1 above)
Comment (diana-macrometa) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: It's pretty weird and redundant, but I wasn't sure if I would remove something important by changing it. I'll try and clean it up.

Comment (joubert-macrometa) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: What about:

If a waiting room is not configured, the waiting room library returns a header labeled No-Waiting-Room. This signifies that no waiting room has been set up.

**Comment by diana-macrometa** [comment]
@joubert-macrometa and @rahul-naik1, please review.

File: docs/photoniq/vwrs/troubleshoot-vwrs.md (hunk 2 above)
Comment (rahul-naik1) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: The cloud origin path is a cloud storage path (e.g. NetStorage, S3 etc) where the waiting room HTML page is stored.
Can we connect to discuss this?

File: docs/photoniq/vwrs/troubleshoot-vwrs.md (hunk 3 above)
Comment (rahul-naik1) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: @joubert-macrometa I am not sure. I will check with either @dwivedianurag or @naimishvora-macrometa for this.

Comment (dwivedianurag) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: @rahul-naik1 @joubert-macrometa the variable name should be encryptionKey and it is set as it is being set earlier (i.e before the npm package ) the only change with npm package is that how we import the package in edgeworker and nothing else is changed. This means the way like we set the configuration option earlier(before npm package) we use that now as well

File: docs/photoniq/vwrs/troubleshoot-vwrs.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+This page provides troubleshooting information for Macrometa PhotonIQ Virtual Waiting Rooms (VWRs).
+
+## No Waiting Room Configured
+
+If a waiting room has not been configured, then the waiting room library returns the header _No-Waiting-Room_. This means that the domain URL is not configured correctly.
+
+## Troubleshoot Waiting Room HTML
+
+You can configure the cloud origin (e.g. NetStorage) path to the waiting room HTML through the library or the domain record. To preview this HTML, open your preferred browser and navigate to the following URL: `https://<hostname>/<waiting-room-path>/preview`
```
Comment (joubert-macrometa) [inline docs/photoniq/vwrs/troubleshoot-vwrs.md]:
```suggestion
You have the option to set the URL for the waiting room HTML template either via the library or the domain record. To get a preview of the waiting room HTML template, launch your preferred web browser and go to the following URL: https://<hostname>/waiting-room-path>/status/preview.
```

File: docs/photoniq/vwrs/troubleshoot-vwrs.md (hunk 2 above)
Comment (diana-macrometa) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: IS this sorted out?

File: docs/photoniq/vwrs/troubleshoot-vwrs.md (hunk 3 above)
Comment (joubert-macrometa) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: Nevermind.  Don't reference encryption key.

**Review by joubert-macrometa — APPROVED** [review]
@diana-macrometa one change here.  But approving so you can finish this.

File: docs/photoniq/vwrs/troubleshoot-vwrs.md (hunk 3 above)
Comment (diana-macrometa) [reply docs/photoniq/vwrs/troubleshoot-vwrs.md]: Do I need to remove it?

**Pull request merged**